src/commands/uplink/index.ts

```diff
@@ -12,3 +12,4 @@ export {default as ExAbsHourMul} from './ExAbsHourMul.js';
 export {default as GetCurrentMul} from './GetCurrentMul.js';
 export {default as SetTime2000} from './SetTime2000.js';
 export {default as Time2000} from './Time2000.js';
+export {default as NewEvent} from './NewEvent.js';
```

Comment:
it will be rather big list

let's keep it sorted

Reply:
fixed